src/main/resources/functions/component.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ ~  Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com).
+ ~
+ ~  WSO2 LLC. licenses this file to you under the Apache License,
+ ~  Version 2.0 (the "License"); you may not use this file except
+ ~  in compliance with the License.
+ ~  You may obtain a copy of the License at
+ ~
+ ~    http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~  Unless required by applicable law or agreed to in writing,
+ ~  software distributed under the License is distributed on an
+ ~  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ ~  KIND, either express or implied.  See the License for the
+ ~  specific language governing permissions and limitations
+ ~  under the License.
+-->
+<!-- Generated on $timestamp -->
+
+<component name="functions" type="synapse/template">
+    <subComponents>
+        <component name="getSchema">
+            <file>getSchema.xml</file>
+            <description>This is grpc GetSchema operation.</description>
```

Comment:
Shall we add a displayName field as well? Similar to https://github.com/wso2-extensions/esb-connector-email/blob/master/src/main/resources/functions/component.xml#L32